templates/cisco_wlc_ssh_show_radius_summary.textfsm

```diff
@@ -0,0 +1,59 @@
+Value Filldown VENDOR_ID_BWD_COMPAT (\S+)
+Value Filldown CALL_STATION_ID_CASE (\S+)
+Value Filldown ACCT_CALL_STATION_ID_TYPE (.+)
+Value Filldown AUTH_CALL_STATION_ID_TYPE (.+)
+Value Filldown EXT_SRC_PORTS_SUPPORT (\S+)
+Value Filldown AGGRESSIVE_FAILOVER (\S+)
+Value Filldown KEYWRAP (\S+)
+Value Filldown FALLBACK_TEST_MODE (\S+)
+Value Filldown FALLBACK_TEST_USERNAME (.+)
+Value Filldown FALLBACK_TEST_INTERVAL (\d+)
+Value Filldown MAC_DELIM_AUTHC_MSG (\S+)
+Value Filldown MAC_DELIM_ACCT_MSG (\S+)
+Value Filldown AUTHC_FRAMED_MTU (\d+)
+Value Required INDEX (\d+)
+Value AUTHC_SRV_ADDR (\S+)
+Value ACCT_SRV_ADDR (\S+)
+Value TYPE (\S+)
+Value SRV_PORT (\d+)
```

Comment:
I'm on the fence, but thinking these may not need the `SRV_` prefix.

Reply:
Removed SRV_ from several variables.